common/static/coffee/spec/discussion/discussion_spec_helper.coffee

```diff
@@ -37,560 +37,34 @@ class @DiscussionSpecHelper
         )
 
     @setUnderscoreFixtures = ->
-        for templateName in ['thread-show']
-            templateFixture = readFixtures('common/templates/discussion/' + templateName + '.underscore')
+        templateNames = [
+            'thread', 'thread-show', 'thread-edit',
+            'thread-response-show', 'thread-response-edit',
```

Comment:
@singingwolfboy  most of the jasmine failures are fixed by adding `'thread-response'` to this array.